commons-lang/src/main/java/org/apache/commons/lang3/util/FluentBitSet.java

```diff
@@ -127,7 +127,7 @@ public BitSet bitSet() {
      * @return the number of bits set to {@code true} in this {@link BitSet}.
      */
     public int cardinality() {
-        return bitSet.cardinality();
+        return (int)bitSet.cardinality();    
```

Comment:
remove trailing spaces but also changes the code, needs more review

Reply:
@rohitmaharjan170 please revise his in 2-3 days, or we'll assign to someone else.